src/odemis/model/_metadata.py

```diff
@@ -134,6 +134,9 @@
 MD_HFW_SLOPE = "HFW slope"  # (float, float) HFW related SEM image shift, slope of linear fit
 MD_SPOT_SHIFT = "Spot shift"  # (float, float), SEM spot shift in percentage of HFW
 
+# The following metadata is for correction on the Nikon C
+MD_SHIFT_LOOKUP = "Pixel shift compensation LUP"
```

Comment:
I'd name it "Pixel shift compensation table"
Also add info on the type in a comment, eg `# dict (int (resolution X) -> dict (float (dwell time) -> tuple of 4 floats (correction factors)))